astropy/io/misc/tests/test_pickle_helpers.py

```diff
@@ -14,17 +16,19 @@ def test_fnpickling_simple(tmp_path):
     fn = str(tmp_path / "test1.pickle")
 
     obj1 = "astring"
-    fnpickle(obj1, fn)
-    res = fnunpickle(fn, 0)
-    assert obj1 == res
-
-    # now try with a file-like object instead of a string
-    with open(fn, "wb") as f:
-        fnpickle(obj1, f)
-    with open(fn, "rb") as f:
-        res = fnunpickle(f)
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=AstropyDeprecationWarning)
```

Comment:
Shouldn't this be

```suggestion
    with pytest.warns(AstropyDeprecationWarning, match="Use pickle from standard library"):
```

Here and elsewhere?

Reply:
Changed and credited you as co-author. Thanks!